docs/tutorials/index.md

```diff
@@ -1,84 +1,319 @@
+</div>
+
+
+<div class="basic-networks">
+
+- [Simple autograd example](http://mxnet.incubator.apache.org/tutorials/gluon/autograd.html)
 - [Automatic differentiation with autograd](http://gluon.mxnet.io/chapter01_crashcourse/autograd.html)
+- [Neural network building blocks with gluon](http://mxnet.incubator.apache.org/tutorials/gluon/gluon.html)
+- [Hybrid network example](http://mxnet.incubator.apache.org/tutorials/gluon/hybrid.html)
+</div>
+
+<div class="linear-regression">
+
 - [Linear regression with gluon](http://gluon.mxnet.io/chapter02_supervised-learning/linear-regression-gluon.html)
-- [Serialization - saving, loading and checkpointing](http://gluon.mxnet.io/chapter03_deep-neural-networks/serialization.html)
+</div>
+
+</div> <!--end of beginner-->
+
+
+<div class="intermediate">
+
 
-### Neural Networks
+<div class="image-recognition">
 
+- [Handwritten digit recognition (MNIST)](http://mxnet.incubator.apache.org/tutorials/gluon/mnist.html)
 - [Multilayer perceptrons in gluon](http://gluon.mxnet.io/chapter03_deep-neural-networks/mlp-gluon.html)
-- [Convolutional Neural Networks in gluon](http://gluon.mxnet.io/chapter04_convolutional-neural-networks/cnn-gluon.html)
-- [Recurrent Neural Networks with gluon](http://gluon.mxnet.io/chapter05_recurrent-neural-networks/rnns-gluon.html)
+- [Convolutional Neural Networks (CNNs) in gluon](http://gluon.mxnet.io/chapter04_convolutional-neural-networks/cnn-gluon.html)
+- [Multi-class object detection using CNNs in gluon](http://gluon.mxnet.io/chapter04_convolutional-neural-networks/cnn-gluon.html)
+- [Visual question answering in gluon](http://gluon.mxnet.io/chapter08_computer-vision/visual-question-answer.html)
+- [Transferring knowledge through fine-tuning (not hotdog)](http://gluon.mxnet.io/chapter08_computer-vision/fine-tuning.html)
+</div>
 
-### Advanced
 
-- [Plumbing: A look under the hood of gluon](http://gluon.mxnet.io/chapter03_deep-neural-networks/plumbing.html)
+<div class="human-language">
+
+- [Simple Recurrent Neural Networks (RNNs) with gluon](http://gluon.mxnet.io/chapter05_recurrent-neural-networks/simple-rnn.html)
+- [Long Short-Term Memory (LSTM) RNNs with gluon](http://gluon.mxnet.io/chapter05_recurrent-neural-networks/lstm-scratch.html)
+- [Gated Recurrent Unit (GRU) RNNs with gluon](http://gluon.mxnet.io/chapter05_recurrent-neural-networks/gru-scratch.html)
+- [Advanced RNNs with gluon](http://gluon.mxnet.io/chapter05_recurrent-neural-networks/rnns-gluon.html)
+- [Tree LSTM modeling for semantic relatedness](http://gluon.mxnet.io/chapter09_natural-language-processing/tree-lstm.html)
+</div>
+
+
+<div class="recommender-systems">
+
+- [Introduction to recommender systems](http://gluon.mxnet.io/chapter11_recommender-systems/intro-recommender-systems.html)
+- [Matrix factorization in recommendation systems](http://mxnet.incubator.apache.org/tutorials/python/matrix_factorization.html)
```

Comment:
result of Gluon&Beginner&Data Loading include:
```
NDArray in Compressed Sparse Row Storage Format
Sparse Gradient Updates
```
these are not available in gluon yet.